pkgs/development/libraries/linenoise/default.nix

```diff
@@ -0,0 +1,29 @@
+  { stdenv, fetchgit }:
+
+  stdenv.mkDerivation rec {
+    name = "linenoise-${version}";
+    version = "git-2016-09-30";
```

Comment:
Please use the convention `<name>-unstable-<version>` or simply `<name>-<version>`. Note that version strings must begin with a digit, everything before the first `-<digit>` is parsed as part of the name.